src/test/java/hudson/plugins/git/ChangelogToRevOptionsTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2017 Mark Waite.
```

Comment:
Probably best to put your name here rather than mine.  I don't want to claim authorship of this test, though I appreciate very much that you've included tests.